apps/examples/testcase/le_tc/network/tc_net_getsockopt.c

```diff
@@ -247,7 +247,8 @@ void net_getsockopt_main(void)
 
 	tc_net_getsockopt_invalid_filedesc_n();
 	tc_net_getsockopt_optval_n(fd);
-	tc_net_getsockopt_sol_socket_so_acceptconn_p(fd);
+	/* get has lwip_tcp in the accepticonn option and not in set. */
+//	tc_net_getsockopt_sol_socket_so_acceptconn_p(fd);
```

Comment:
you should comment definition also or else build may fail with error or warning depending on -Werror compiler option